python/taichi/lang/ops.py

```diff
@@ -1174,6 +1349,28 @@ def assign(a, b):
 
 
 def max(*args):  # pylint: disable=W0622
+    """Compute the maximum of the arguments, element-wise.
+
+    This function takes no effect on a single argument, even it's array-like.
+    When there are both scalar and matrix arguments in `args`, the matrices
+    must have the same shape, and scalars will be broadcasted to the same shape as the matrix.
+
+    Args:
+        args: (List[:mod:`~taichi.types.primitive_types`, :class:`~taichi.Matrix`]): \
+            The input.
+
+    Returns:
+        Maximum of the inputs.
+
+    Example::
+
+        >>> @ti.kernel
+        >>> def foo():
+        >>>     x = ti.Vector([0, 1, 2])
+        >>>     y = ti.Vector([3, 4, 5])
+        >>>     z = ti.max(x, y, 4)
+        >>>     print(z)  # [4, 4, 5]
```

Comment:
Not sure if really works currently :thinking:

Reply:
You may this behavior has changed recently?

Reply:
Yeah, I haven't use this func in a while so just built Taichi from source, the above code does not work well on the current `master` branch. Maybe I missed something?

Reply:
Per offline discussion, I was using 0.8.8 at this place, NVM.